sdk/ai/ai-voicelive/review/ai-voicelive-node.api.md

```diff
@@ -0,0 +1,1669 @@
+    eventId?: string;
+    itemId: string;
+    type: "conversation.item.deleted";
+}
+
+// @public
+export interface ServerEventConversationItemInputAudioTranscriptionCompleted extends ServerEvent {
+    contentIndex: number;
+    itemId: string;
+    transcript: string;
+    type: "conversation.item.input_audio_transcription.completed";
+}
+
+// @public
+export interface ServerEventConversationItemInputAudioTranscriptionDelta extends ServerEvent {
+    contentIndex?: number;
+    delta?: string;
+    itemId: string;
+    logprobs?: LogProbProperties[];
+    type: "conversation.item.input_audio_transcription.delta";
+}
+
+// @public
+export interface ServerEventConversationItemInputAudioTranscriptionFailed extends ServerEvent {
+    contentIndex: number;
+    error: VoiceLiveErrorDetails;
+    itemId: string;
+    type: "conversation.item.input_audio_transcription.failed";
+}
+
+// @public
+export interface ServerEventConversationItemRetrieved extends ServerEvent {
+    // (undocumented)
+    eventId?: string;
+    // (undocumented)
+    item?: ResponseItemUnion;
+    type: "conversation.item.retrieved";
+}
+
+// @public
+export interface ServerEventConversationItemTruncated extends ServerEvent {
+    audioEndInMs: number;
+    contentIndex: number;
+    // (undocumented)
+    eventId?: string;
+    itemId: string;
+    type: "conversation.item.truncated";
+}
+
+// @public
+export interface ServerEventError extends ServerEvent {
+    error: ServerEventErrorDetails;
+    type: "error";
+}
+
+// @public
+export interface ServerEventErrorDetails {
+    code?: string;
+    eventId?: string;
+    message: string;
+    param?: string;
+    type: string;
+}
+
+// @public
+export interface ServerEventInputAudioBufferCleared extends ServerEvent {
+    type: "input_audio_buffer.cleared";
+}
+
+// @public
+export interface ServerEventInputAudioBufferCommitted extends ServerEvent {
+    itemId: string;
+    previousItemId?: string;
+    type: "input_audio_buffer.committed";
+}
+
+// @public
+export interface ServerEventInputAudioBufferSpeechStarted extends ServerEvent {
+    audioStartInMs: number;
+    itemId: string;
+    type: "input_audio_buffer.speech_started";
+}
+
+// @public
+export interface ServerEventInputAudioBufferSpeechStopped extends ServerEvent {
+    audioEndInMs: number;
+    itemId: string;
+    type: "input_audio_buffer.speech_stopped";
+}
+
+// @public
+export interface ServerEventResponseAnimationBlendshapeDelta extends ServerEvent {
+    // (undocumented)
+    contentIndex: number;
+    // (undocumented)
+    frameIndex: number;
+    // (undocumented)
+    frames: number[][] | string;
+    // (undocumented)
+    itemId: string;
+    // (undocumented)
+    outputIndex: number;
+    // (undocumented)
+    responseId: string;
+    // (undocumented)
+    type: "response.animation_blendshapes.delta";
+}
+
+// @public
+export interface ServerEventResponseAnimationBlendshapeDone extends ServerEvent {
+    // (undocumented)
```

Comment:
I'd love to see these all documented (either now or in a follow-up PR, feel free to make a separate issue)

Reply:
The TSP isn't doc'd so we'll likely handle that as a follow-up.